cmd/queue/main.go

```diff
@@ -114,14 +107,15 @@ var (
 		appResponseTimeInMsecN,
 		"The response time in millisecond",
 		stats.UnitMilliseconds)
+
 	readinessProbeTimeout = flag.Int("probe-period", -1, "run readiness probe with given timeout")
 )
 
 type config struct {
 	ContainerConcurrency         int    `split_words:"true" required:"true"`
 	QueueServingPort             int    `split_words:"true" required:"true"`
 	RevisionTimeoutSeconds       int    `split_words:"true" required:"true"`
-	UserPort                     int    `split_words:"true" required:"true"`
+	UserPort                     string `split_words:"true" required:"true"`
```

Comment:
While this saves an itoa call later, this validates it's not "BAD_PORT".

Reply:
Fair game, will change!